tests/functional/spec/census_exit.spec.js

```diff
@@ -28,6 +28,6 @@ describe("Post submission exit", () => {
     $(CensusThankYouPage.exit()).click();
     browser.back();
     expect(browser.getUrl()).to.contain("submitted/thank-you");
-    expect($("body").getHTML()).to.contain("This page is no longer available");
+    expect($("body").getHTML()).to.contain("Your session has timed out due to inactivity");
```

Comment:
It doesn't make sense to introduce code to determine if something is census (which isn't used any more) based or not on the post_submission session expiration.

Reply:
I think while we are here, we should rename the file to `exit.spec.js` since it is not tied to Census. This behaviour should be the same for all schemas.

Reply:
True